sunbeam-python/sunbeam/features/instance_recovery/feature.py

```diff
@@ -72,6 +81,8 @@ def manifest_attributes_tfvar_map(self) -> dict:
             }
         }
 
+        return consul_tfvar_map | instance_recovery_tfvar_map
```

Comment:
Need a deep copy of the dicts here. Currently consul_tfvar_map is overrided which leads to consul-channel not set in terraform tfvars.